core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1889,9 +1889,13 @@ object SnappySession extends Logging {
     def apply(session: SnappySession, lp: LogicalPlan, sqlText: String, pls:
     ArrayBuffer[ParamLiteral]): CachedKey = {
 
+      val invalidate = Array(false)
```

Comment:
No Sumedh. Vivek meant val vs var only.
BTW I used val because I was toggling the flag inside the normalizeExprIds function inside the apply method. I thought that will be better.